part3/src/main/scala/io/smartdatalake/workflow/dataobject/CustomWebserviceDataObject.scala

```diff
@@ -24,8 +24,8 @@ import scala.util.{Failure, Success}
 
 case class HttpTimeoutConfig(connectionTimeoutMs: Int, readTimeoutMs: Int)
 
-// Default to the interval of 2 days from now to today
-case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis() / 1000 - 172800L, end: Long = System.currentTimeMillis() / 1000)
+// Default to the interval of [2 weeks and 2 days ago] -> [2 weeks ago]
+case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis()/1000 - 1209600 - 172800L, end: Long = System.currentTimeMillis()/1000 - 1209600 )
```

Comment:
Are there any code formatting rules to follow?

```suggestion
case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis() / 1000 - 1209600 - 172800L, end: Long = System.currentTimeMillis() / 1000 - 1209600 )
```